### PR TITLE
Rename 'Ack' messages and logic to 'Processed'

### PR DESCRIPTION
**Pull request by carllin**
For this issue: https://github.com/raiden-network/raiden/issues/1341

### CODE REVIEW

**Comment by LefterisJP** [comment]
@carllin Hello. Thanks for the PR. Can you configure your git (check [these docs for how](https://help.github.com/articles/setting-your-commit-email-address-in-git/)) so that the same name and email as in your github account is the one making the commits?

Also please do not merge master into your branch. Instead squash all of the commits into one and rebase it onto master.

**Comment by carllin** [comment]
@LefterisJP thanks for taking a look! Addressed the issues

File: raiden/network/protocol.py [hunk 1]
```diff
@@ -661,36 +662,40 @@ def send_async(self, receiver_address, message):
...
 
-        messagedata = ack_message.encode()
-        self.receivedhashes_to_acks[ack_message.echo] = (receiver_address, messagedata)
-        self._maybe_send_ack(*self.receivedhashes_to_acks[ack_message.echo])
+        messagedata = processed_message.encode()
 
-    def _maybe_send_ack(self, receiver_address, messagedata):
-        """ ACK must not go into the queue, otherwise nodes will deadlock
+        self.receivedhashes_to_processedmessages[processed_message.echo] = (
+            receiver_address, messagedata)
```
Comment (LefterisJP) [inline raiden/network/protocol.py]: style note here. Change this to:

```python
self.receivedhashes_to_processedmessages[processed_message.echo] = (
            receiver_address,
            messagedata,
)
```

Same with `self._maybe_send_processed` below

File: raiden/network/protocol.py [hunk 2]
```diff
@@ -661,36 +662,40 @@ def send_async(self, receiver_address, message):
...
+
+        self._maybe_send_processed(
+            *self.receivedhashes_to_processedmessages[processed_message.echo])
+
+    def _maybe_send_processed(self, receiver_address, messagedata):
+        """ `Processed` messages must not go into the queue, otherwise nodes will deadlock
         waiting for the confirmation.
         """
```
Comment (LefterisJP) [inline raiden/network/protocol.py]: style here: Closing `"""` should not go on its own line.

File: raiden/network/protocol.py (hunk 1 above)
Comment (carllin) [reply raiden/network/protocol.py]: Made the fixes

**Pull request merged**